sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -458,34 +458,48 @@ def connect_channel(channel, stub, port):
         else:
             break
 
-def create_channel(type_chan, level, kvp, soc_ip, port, asic_index, fwd_state_response_tbl, is_async):
+def wait_for_state_change(channel_connectivity, port):
+    # Initialize YcableChannelStateTableHelper only once
+    if not hasattr(wait_for_state_change, 'table_helper'):
+        wait_for_state_change.table_helper = {}
```

Comment:
@vdahiya12 please use Singleton pattern for single instance of `YcableChannelStateTableHelper`

Reply:
same comment as below for this. 
As channel.subscribe launches into a new separate thread for every single port, its better to not share this across multiple ports, or else we run into same problem